playbooks/nfs.yml

```diff
@@ -0,0 +1,107 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Deploy an NFS server
+  become: true
+  hosts: "{{ groups[cifmw_nfs_target | default('computes')][0] | default([]) }}"
+  pre_tasks:
+    - name: End play early if no NFS is needed
+      when:
+        - not cifmw_edpm_deploy_nfs | default('false') | bool
+      ansible.builtin.meta: end_play
+  tasks:
+    - name: Install nfs-utils package
+      ansible.builtin.package:
+        name: "nfs-utils"
+
+    - name: Configure nfs to use v4 only
+      community.general.ini_file:
+        path: /etc/nfs.conf
+        section: nfsd
+        option: vers3
+        value: n
```

Comment:
i was mainly curious about the value of 'n' for this but it doesn't seem like a typo

So I guess, vers3 includes ipv6 so disabling it means we only get v4? At first I thought this was about IPv4 but probably it is about the version of NFS protocol? 

(reference for others https://www.man7.org/linux/man-pages/man8/rpc.nfsd.8.html)

Reply:
that's right, we only want v4, so this option disables v3. Openstack components only support v4, that's the underlying reason for this config